.github/ISSUE_TEMPLATE/---found-a-bug-and-i-solved-it.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F41E Found a Bug!"
+about: I found a problem! It is Jina’s bug and I’d like to bring it to others' attention.
+title: ''
+labels: kind/bug
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**Describe how you solve it**
```

Comment:
"Have you solved it? If so, let us know how!" - bc I guess many *haven't* solved it if they're posting an issue

Reply:
Changed.